.circleci/config.yml

```diff
@@ -18,9 +18,7 @@ jobs:
           name: Install Python dependencies
           command: |
             pip --version
-            pip install --progress-bar off . \
-              -r ./dev/large-requirements.txt \
-              sphinx sphinx-click
+            pip install --progress-bar off -r dev-requirements.txt .
```

Comment:
Instead of manually specifying `sphinx` and `sphinx-click`, use `dev-requirements.txt`.